Automattic/wp-calypso
# Sensei Flow: Update checkout summary features list

Resolves https://github.com/Automattic/wp-calypso/issues/85803

## Proposed Changes

* Updates the features list on the checkout summary to be the same as the plan selection features list.

## Testing Instructions



* Go to http://calypso.localhost:3000/setup/sensei/
* Select a domain and reach plan selection.
* Make sure the feature list on the monthly/yearly plans is the same as the one on the checkout.

Monthly features when selecting the plan:

![image](https://github.com/Automattic/wp-calypso/assets/1612178/1a9837f8-4ea4-4290-ae68-7348c1de3415)

Monthly features on checkout:

![image](https://github.com/Automattic/wp-calypso/assets/1612178/83b434c7-b7cd-44d2-9264-fbc850a44c3a)

Yearly features when selecting the plan:

![image](https://github.com/Automattic/wp-calypso/assets/1612178/68db3ffe-ff95-426c-800c-49f2118a8262)

Yearly features on checkout:

![image](https://github.com/Automattic/wp-calypso/assets/1612178/713b4e2e-d8c2-46d1-8c5b-5c3e115d5b6d)


## Pre-merge Checklist
